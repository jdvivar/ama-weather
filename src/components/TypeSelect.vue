<template>
  <div class="line --interactive">
    <router-link to="/">Pro</router-link>
    <router-link to="/pro">Basic</router-link>
  </div>
</template>
<script>
export default {
  name: 'type-select'
}
</script>

<style scoped>
.router-link-exact-active {
  display: none;
}

a {
  color: inherit;
  text-decoration: none;
}
</style>
