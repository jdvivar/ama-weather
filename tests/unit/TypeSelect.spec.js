import { shallowMount } from '@vue/test-utils'
import TypeSelect from '@/components/TypeSelect'

describe('TypeSelect', () => {
  it('has correct markup', () => {
    const wrapper = shallowMount(TypeSelect)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
