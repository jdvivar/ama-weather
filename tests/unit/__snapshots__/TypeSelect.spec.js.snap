// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TypeSelect has correct markup 1`] = `
<div class="line --interactive">
  <router-link-stub to="/">Pro</router-link-stub>
  <router-link-stub to="/pro">Basic</router-link-stub>
</div>
`;
